.circleci/config.yml

```diff
@@ -5,9 +5,10 @@ executors:
   android-executor:
     docker:
       - image: circleci/android:api-28
+    resource_class: medium
     working_directory: ~/purchases-android
     environment:
-      JVM_OPTS: -Xmx3200m
+      JVM_OPTS: -Xmx4096m
```

Comment:
did increasing the workers remove the need for this?

Reply:
I think it was a combination of both: increasing workers, no-daemon, more memory and fixing that issue with Jacoco